tirocks/src/statistics.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::{fmt::Display, mem::MaybeUninit};
+
+use tirocks_sys::{
+    rocksdb_HistogramData, rocksdb_Histograms, rocksdb_Tickers, rocksdb_titandb_HistogramType,
+    rocksdb_titandb_TickerType,
+};
+
+use crate::util;
+
+/// A trait for marking ticker types.
+pub trait TickerType {
+    /// The raw ID of the ticker.
+    fn r#type(&self) -> u32;
```

Comment:
Looks a bit disturbing.

Reply:
Why?

Reply:
Unfamiliar to the fingers, the eyes, and IDE.. I think raw_type is fine unless "#" bears a particular meaning

Reply:
Oh, you mean `r#`. It's rust convention to escape keyword.

Reply:
Okay, TIL